.github/workflows/deploy_tre_reusable.yml

```diff
@@ -252,11 +252,11 @@ jobs:
         if: steps.ci_cache_cr_login.outcome != 'success'
         run: |
           # shellcheck disable=SC2034,SC2015,SC2125
-          for i in {1..3}; do
-            az acr login --name "${{ secrets.CI_CACHE_ACR_NAME }}" && ec=0 && break || ec=\$? && sleep 10
+          for i in {1..4}; do
+            az acr login --name "${{ secrets.CI_CACHE_ACR_NAME }}" && ec=0 && break || ec=\$? && sleep 60
```

Comment:
Wouldn't it make more sense to add more retries and not increase the time between so much?